core/templates/pages/classroom-page/classroom-page.component.css

```diff
@@ -72,27 +72,26 @@
 
 .oppia-classroom-viewer-container .oppia-topic-summary-tiles {
   display: grid;
-  grid-gap: 2vw;
+  gap: 24px;
   grid-template-columns: 210px 210px 210px;
-  justify-content: space-between;
+  justify-content: center;
   margin-top: 40px;
   overflow-y: auto;
   padding-bottom: 18vh;
 }
 
-.oppia-classroom-viewer-container .oppia-topic-summary-tile {
-  padding-bottom: 40px;
-}
-
 .oppia-classroom-viewer-container .classroom-name {
   font-size: 3em;
 }
 .oppia-classroom-viewer-container .classroom-header {
   font-size: 1.2em;
 }
 .oppia-classroom-viewer-container .oppia-classroom-banner {
-  margin-left: 25vw;
   margin-top: -120px;
+  text-align: center;
+}
+.oppia-classroom-banner img {
```

Comment:
Here and below, instead of specifying "img" can we use a class name instead?

Reply:
Addressed it in the most recent commit! PTAL